packages/preview/glossarium-0.1.0/typst.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "glossarium"
+version = "0.1.0"
+entrypoint = "glossarium.typ"
+authors = ["slashformotion", "Dherse"]
+license = "MIT"
+description = "Glossarium is a simple, easily customizable typst glossary."
+repository = "https://github.com/slashformotion/glossarium"
+exclude = ["LICENCE", "example/*"]
```

Comment:
I don't recommend excluding the license. It doesn't increase the package size by much and it is better if it always shipped with the code.

Reply:
changed